docs/content/docs/tutorials/emr-on-eks-example.md

```diff
@@ -0,0 +1,494 @@
+---
+title: "Run Spark jobs using EMR on EKS controller"
+description: "ACK service controller for EMR on EKS enables customers to run spark jobs on EKS clusters"
+lead: "Run Spark jobs using ACK service controller for EMR on EKS."
+draft: false
+menu:
+  docs:
+    parent: "tutorials"
+weight: 47
+toc: true
+---
+We are excited to announce ACK service controller for EMR on EKS. Customers have told us that they like the declarative way of managing resources on EKS clusters. Using ACK service controller for EMR on EKS, customers now have the ability to define and run EMR jobs directly from their Kubernetes clusters. EMR on EKS manages the lifecycle of these jobs and it is [3.5 times faster than open-source Spark](https://aws.amazon.com/blogs/big-data/amazon-emr-on-amazon-eks-provides-up-to-61-lower-costs-and-up-to-68-performance-improvement-for-spark-workloads/) because it uses highly optimized EMR runtime  
+
+To get started, you can download the EMR on EKS controller image from [Amazon ECR](https://gallery.ecr.aws/aws-controllers-k8s/emrcontainers-controller) and run Spark jobs in minutes. ACK service controller for EMR on EKS is available as a *developer preview* and is not recommended for production use. To learn more, visit the [EMR on EKS documentation](https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/emr-eks.html).
```

Comment:
Just spoke internally with an ACK core member and we agreed that we'd like to not include Press Release like statements in the blog posts. Ideally this blog post should be similar to all the other ones we have.

Reply:
sure thing. I'll remove the PR statements in next commit